build.gradle

```diff
@@ -43,15 +52,23 @@ subprojects {
     dependencyManagement {
         dependencies {
             dependency 'commons-logging:commons-logging:1.2'
+            dependency 'org.springframework.boot:spring-boot-starter:2.0.0.M3'
         }
     }
 
-    repositories {
-        jcenter()
+    dependencies {
+        testCompile('junit:junit')
+        testCompile('org.mockito:mockito-core')
+        testCompile('org.assertj:assertj-core')
+        testCompile('org.springframework.boot:spring-boot-starter-test')
+        testCompile('commons-logging:commons-logging')
```

Comment:
- Removed the trans. deps.
- I just included it because there were so many subprojects explicitly requiering it and i thought you realy want this logging impl